### ISSUE TITLE
Support for nested object in node state

### ISSUE THREAD

**Issue by silentninja** [schibsted/layout#82]
When setting a node's state using an object, we can only reference that object's simple built in datatypes, would be really nice if we can get a reference the properties of the objects inside the object.

```
class NestedModel {
var name: String?;
}
class Model {
var nestedModel: NestedModel;
}
```
in the layout would like to access it like
```
    <UILabel
        width="100%"
        textColor="titleColor"
        text="{nestedModel.name}"
    />
```

**Comment by nicklockwood** [comment]
I'll see if I can add support for this.

**Comment by mariob** [comment]
Yeah. Did tinkering with a settings screen and had to prefix a lot of properties in my struct instead of using nested structs. Hope this is possible to do.

**Comment by nicklockwood** [comment]
This is now implemented on the develop branch.

**Comment by nicklockwood** [comment]
Implemented in 0.6.8
